package.json

```diff
@@ -11,7 +11,8 @@
   },
   "scripts": {
     "start": "gulp",
-    "build": "gulp sass"
+    "build": "gulp sass",
+    "test": "bundle exec jekyll serve --no-watch"
```

Comment:
This is a good idea, thanks! However I'd rather not overload `test`, as I'd like (one day) to actually have some tests we can run here. Perhaps naming it `jekyll` would be best? 

Aside: I tried to find out if there were any naming conventions here, it seems `start` is most common, but that should build and run everything. Perhaps we can add that to combine `build` and `jekyll`